articles/help/toc.yml

```diff
@@ -0,0 +1,4 @@
+- name: FAQ
+  href: faq.md
+- name: Github
+  href: github.md
```

Comment:
For maintainability purposes, is it possible for the `name`s to be blank and have DocFX automatically populate it with the article title from an `xref`? This may require that we change TOC files from `.yml` to `.md`.

Reply:
I wish. Until we can figure that out, I don't think there is a way to do that

Reply:
I wish @JonathanLydall . It seems like DocFX only supports `md` and `yml` file formats but with a Table Of Contents structure for each type. Once could perhaps log a feature request on their Github page.

Reply:
Are you saying @JonathanLydall  that we can let DocFX pull the names from the actual MD file's header into the TOC by leaving the `name` fields blank? I quickly tried it out but it seems to generate `[](faq.md)` and `[](github.md)` instead. Maybe everything has to be MD based instead of YAML based? Not sure. We could go and log a feature request on DocFx's site.

Reply:
Does the trick with it automatically filling in the title work with normal articles, if not then we could try update DocFX.

If it's not due the DocFX version, then let's no worry about the issue in TOC files.